pkgs/applications/window-managers/xborders/default.nix

```
@@ -0,0 +1,39 @@
+{ lib, python3Packages, fetchFromGitHub, libwnck, gtk3, libnotify, wrapGAppsHook, gobject-introspection }:
+
+with python3Packages; buildPythonPackage rec {
```

```suggestion
buildPythonPackage rec {
```

Do not abuse `with`.